aspnetcore/blazor/get-started.md

```diff
@@ -74,25 +74,25 @@ Get started with Blazor:
 
    7\. In a browser, navigate to `https://localhost:5001`.
 
-   <!--
-
    # [Visual Studio for Mac](#tab/visual-studio-mac)
 
    1\. Install [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/). Switch the [Update channel to Preview](/visualstudio/mac/install-preview).
```

Comment:
Blazor Server support will move into the stable channel next week.

Reply:
I figured we'd make that change (strip that) when the time comes, but do you want to hold this PR for a week and strip it now?